js/gated-content/src/views/VirtualMeetingListingPage.vue

```diff
@@ -19,5 +19,11 @@ export default {
   components: {
     EventListing,
   },
+  computed: {
+    emptyMsg() {
+      return this.config.virtual_meeting_empty_text === ''
```

Comment:
field is required, so no need for a ternary operator, let's rely on having the default value

Reply:
@podarok I think, it is better to have this check on the frontend since every Y could alter the form and make a field not required.